app/views/layouts/application.html.erb

```diff
@@ -1,16 +1,25 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>LittleEstyShop7</title>
+    <title>Little Esty Shop</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
-    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_importmap_tags %>
   </head>
-
   <body>
-    <%= yield %>
+    <h1 class="main_title">Welcome to Little Esty Shop</h1>
+    <div class="merchant_title">
+      <h2>Ray's Handmade Jewelry</h2>
+      <nav class="main-menu">
+        <%= link_to "Dashboard", root_path %>
+        <%= link_to "My Items", root_path %>
+        <%= link_to "My Invoices", merchant_invoices_path(@merchant) %>
+      </nav>
+    </div>
+    <div class="seperator"></div>
+    <div class="container">
+      <%= yield %>
+    </div>
```

Comment:
This looks good, but will cause everything to appear on all pages across the app and these nav links are not relevant for the Admin pages.

Can you move this over into a partial or your view's direct file instead?

Reply:
Also, I believe Dani's User Stories include creating these nav links for the merchant pages. So you might want to double check with her on this aspect

Reply:
@westonio I made the requested changes regarding the navbar. The CSS remains to be implemented sitewide later.

Reply:
Thank you kindly @MWoodshop ! looks perfect